tools/src/faircv/Options.hs

```diff
@@ -0,0 +1,68 @@
+module Options
+       ( FaircvOptions(..)
+       , getFaircvOptions
+       ) where
+
+import Options.Applicative (Parser, auto, execParser, fullDesc, help, helper,
+                            info, long, metavar, option, progDesc, short,
+                            showDefault, strOption, value)
+import Time.Rational (RatioNat)
+
+
+data FaircvOptions = FaircvOptions
+    { witnessUrl    :: String
```

Comment:
What do you think about the following:
Make this parameter optional. In `node.sh` and `demo.sh` make `witness-listen` and `educator-listen` parameters have the same value - this will effectively make educator node serve both witness and educator APIs on the same port. And from this executable, you request `educatorUrl` when `witnessUrl` is unspecified.

Reply:
I liked the idea, but it seems we cannot do that. I just tried and it seems that communications using the `StudentApiClient` result all in a 404.

Reply:
Weird, I have run the executable like `stack exec dscp-faircv localhost:8091 localhost:8090 ./run/tmp/educator.key` and that was working.
But ok, that's not significant.

Reply:
well, but `8090` and `8091` are different, maybe I didn't understand your suggestion

Reply:
Generally, you need no cluster of witnesses, having a single educator (which is also a witness) is enough. So you can replace `8091` with `8090` [here](https://github.com/DisciplinaOU/disciplina/blob/8032e15454ab099030dfc9d83c6e8e2c1f6b1be1/scripts/launch/node.sh#L54), and launch

```
> ./scripts/launch/node.sh e b
> stack exec dscp-faircv -- -w localhost:8090 -e localhost:8090 -k ./run/tmp/educator.key
``` 
Now `-w` and `-e` accept the same value and it works

Although you may have reasons to launch separate witnesses especially since that discovered a bug.

Reply:
Oh ok, I understood you then, but I have tried with a cluster (unsuccessfully).
I'll give it another try later.